### DISCUSSION TITLE
Open Github Sponsor for jbang project

### DISCUSSION THREAD

**Discussion by linux-china** [jbangdev/jbang#1130] [General]
Any plan to open Github Sponsor for jbang project?  I will be first one, and just let you guys have a coffee during contribution.

**Comment by maxandersen** [comment]
Thanks for the sentiment. I'm torn on how/if to open a sponsor/collective as i'm personally paid professionally to work on other opensource projects.

That said I do like the idea of providing the option and thus I requested a open collective and github sponsor account for jbangdev for the intent to use any money this brings to any possible future meetup and/or contribute to organizations/events that promotes/educates around making Java more accessible.

I'll post here once its approved (its pending atm)
